File: vargplvm/matlab/rbfard2VardistPsi0Gradient.m
function [gKern, gVarmeans, gVarcovars] = rbfard2VardistPsi0Gradient(rbfard2Kern, vardist, covGrad)

% RBFARD2VARDISTPSI0GRADIENT Description
  
% VARGPLVM
gKern = zeros(1,rbfard2Kern.nParams); 
gKern(1) = covGrad*vardist.numData;
 
gVarmeans = zeros(1,prod(size(vardist.means))); 
gVarcovars = zeros(1,prod(size(vardist.means))); 

File: vargplvmR/R/rbfard2VardistPsi0Gradient.R
rbfard2VardistPsi0Gradient <-
function (rbfard2Kern, vardist, covGrad)
{
# % RBFARD2VARDISTPSI0GRADIENT  description not available.  
# % FORMAT
# % DESC 
# % description not available

  gKern <- matrix(0, 1, rbfard2Kern$nParams)  
  gKern[1] <- covGrad*vardist$numData 
  
  gVarmeans <- matrix(0, 1, prod(dim(vardist$means)))  
  gVarcovars <- matrix(0, 1, prod(dim(vardist$means)))  
  return(list(gKern = gKern, gVarmeans = gVarmeans, gVarcovars = gVarcovars))
}
